service/systemd/service.go

```diff
@@ -55,24 +55,29 @@ func ListCommand() string {
 	return cmds.listAll()
 }
 
+// Type alias for a DBusAPI factory method.
+type DBusFac = func() (DBusAPI, error)
```

Comment:
Can we please use Factory rather than just Fac?

Reply:
Sure.